script.js

```diff
@@ -10,60 +10,61 @@ play.style.display = "flex";
 main.style.display = "none";
 
 //Enter PLAY mode
-play.addEventListener("click", (e) => {
+play.addEventListener("click", () => {
   play.style.display = "none";
   main.style.display = "flex";
 });
 
 // Hand events
-rock.addEventListener("click", (e) => {
-  if (handIndex().win === "rock") {
-    console.log("TIE");
-  } else if (handIndex().win === "paper") {
-    console.log("YOU LOSE");
-  } else if (handIndex().win === "scissor") {
-    console.log("YOU WIN");
-  }
-});
 
-paper.addEventListener("click", (e) => {
-  if (handIndex().win === "paper") {
-    console.log("TIE");
-  } else if (handIndex().win === "scissor") {
-    console.log("YOU LOSE");
-  } else if (handIndex().win === "rock") {
-    console.log("YOU WIN");
-  }
+rock.addEventListener("click", () => {
+  const ROCK_HAND = {
+    rock: "TIE",
+    paper: "YOU LOSE",
+    scissor: "YOU WIN",
+  };
+  const rockResult = ROCK_HAND[handIndex().win];
+  console.log(rockResult);
 });
 
-scissor.addEventListener("click", (e) => {
-  if (handIndex().win === "scissor") {
-    console.log("TIE");
-  } else if (handIndex().win === "rock") {
-    console.log("YOU LOSE");
-  } else if (handIndex().win === "paper") {
-    console.log("YOU WIN");
-  }
+paper.addEventListener("click", () => {
+  const PAPER_HAND = {
+    rock: "YOU WIN",
+    paper: "TIE",
+    scissor: "YOU LOSE",
+  };
+  const paperResult = PAPER_HAND[handIndex().win];
+  console.log(paperResult);
 });
 
-//Hand details
-rock = {
-  active: true,
-  win: "scissor",
-  emoji: "🗿",
-};
-paper = {
-  active: true,
-  win: "rock",
-  emoji: "📄",
-};
-scissor = {
-  active: true,
-  win: "paper",
-  emoji: "✂️",
-};
+scissor.addEventListener("click", () => {
+  const SCISSOR_HAND = {
+    rock: "YOU LOSE",
+    paper: "YOU WIN",
+    scissor: "TIE",
+  };
+  const scissorResult = SCISSOR_HAND[handIndex().win];
+  console.log(scissorResult);
+});
 
-const hand = [rock, paper, scissor];
+//Hand object
+const hand = [
+  (rock = {
+    active: true,
+    win: "scissor",
+    emoji: "🗿",
+  }),
+  (paper = {
+    active: true,
+    win: "rock",
+    emoji: "📄",
+  }),
+  (scissor = {
+    active: true,
+    win: "paper",
+    emoji: "✂️",
+  }),
+];

```

Comment:
Even tho this might work, it's not ok

You can approach the array in two ways:

```suggestion
const hand = [
  {
    active: true,
    win: "scissor",
    emoji: "🗿",
  },
  {
    active: true,
    win: "rock",
    emoji: "📄",
  },
  {
    active: true,
    win: "paper",
    emoji: "✂️",
  },
];
```

And then you can use each one referring them by the index each option is in.

The other way is to do it in an object, instead of an array

```suggestion
const hand = {
  rock: {
    active: true,
    win: "scissor",
    emoji: "🗿",
  },
  paper: {
    active: true,
    win: "rock",
    emoji: "📄",
  },
  scissor: {
    active: true,
    win: "paper",
    emoji: "✂️",
  },
];
```